composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Better Reflection - an improved code reflection API",
     "license": "MIT",
     "require": {
-        "php": ">=7.2.0,<7.5.0",
+        "php": ">=7.3.0,<7.5.0",
```

Comment:
PHP 7.2.0 is gone: it isn't supported by `infection/infection`, so I'd rather have it gone, and have this target `4.4.0` of this repo instead.